content/en/users/workload-manager/_index.md

```diff
@@ -4,9 +4,463 @@ linkTitle: "Workload Manager"
+Creating proxy for glarocca@wenmr_user
+(/DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Giuseppe La Rocca larocca@egi.eu)
+Requested adding a VOMS extension but no VOMS attribute defined for group wenmr_user
+Uploading proxy for wenmr_user...
+Uploading wenmr_user proxy to ProxyManager...
+Loading user proxy Uploading proxy on-the-fly
+Cert file /home/larocca/.globus/usercert.pem Key file
+/home/larocca/.globus/userkey.pem
+Loading cert and key User credentials loaded
+Uploading...  Proxy uploaded Proxy generated:
+subject      : /DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Giuseppe La Rocca larocca@egi.eu/CN=5721279051
+issuer       : /DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Giuseppe La Rocca larocca@egi.eu
+identity     : /DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Giuseppe La Rocca larocca@egi.eu
+timeleft     : 23:59:58
+DIRAC group  : wenmr_user
+rfc          : True
+path         : /tmp/x509up_u0
+username     : glarocca
+properties   : LimitedDelegation, GenericPilot, Pilot, NormalUser
+
+Proxies uploaded: DN
+| Group               | Until (GMT) /DC=org/DC=terena/DC=tcs/C=NL/O=Stichting
+EGI/CN=Giuseppe La Rocca larocca@egi.eu | access.egi.eu_user  | 2021/09/14 23:54
+/DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Giuseppe La Rocca
+larocca@egi.eu | fedcloud_user       | 2021/09/14 23:54
+/DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Giuseppe La Rocca
+larocca@egi.eu | access.egi.eu_admin | 2021/09/14 23:54
+/DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Giuseppe La Rocca
+larocca@egi.eu | wenmr_user          | 2021/09/14 23:54
+```
+
+As a result of this command, several operations are accomplished:
+
+* A long user proxy (with the length of the validity of the certificate) is
+uploaded to the DIRAC ProxyManager service, equivalent of the gLite MyProxy
+service;
+* A short user proxy is created with the DIRAC extension carrying the DIRAC
+group name and with the VOMS extension corresponding to the DIRAC group if the
+gLite UI environment is available.
+
+If the gLite UI environment is not available, the VOMS extensions will not be
+loaded into the proxy.  This is not a serious problem, still most of the
+operations will be possible.
+
+For checking the details of you proxy, run the following command:
+
+```sh
+$ dirac-proxy-info
+subject      : /DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Giuseppe La Rocca larocca@egi.eu/CN=5721279051
+issuer       : /DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Giuseppe La Rocca larocca@egi.eu
+identity     : /DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Giuseppe La Rocca larocca@egi.eu timeleft     : 23:59:26
+DIRAC group  : wenmr_user
+rfc          : True
+path         : /tmp/x509up_u0
+username     : glarocca
+properties   : LimitedDelegation, GenericPilot, Pilot, NormalUser
+```
+
+#### Managing simple jobs
+
+In the following table there is a comparison between the glite-WMS CLI and the
+DIRAC one:
+
+| gLite-WMS commands | DIRAC commands | Note |
```

Comment:
WMS is long gone, do we really need to compare with it? I would just list the DIRAC commands and what they do

Reply:
I was also doubtful about the comparison with old commands. I'm too not not sure it's worth to mention this at all: we present how to use our service, its command and voilà.

Reply:
Let's drop this table

Reply:
fd149baa47d85da3e7c988861dbe1c902fb2254e
Remove the  gLite-WMS commands, should we drop the entire table?

Reply:
I think @glarocca agreed to do so.